srcalc/src/test/java/gov/va/med/srcalc/domain/CalculationTest.java

```diff
@@ -18,7 +18,7 @@
 {
     protected Patient dummyPatient()
     {
-        return new Patient(1, "Zach Smith");
+        return new Patient(1, "Zach Smith", "M", 40);
```

Comment:
Why don't we move this to SampleObjects now. That way both this test and MockVistaPatientDao can share the same code.